frontend/src/components/landingPage/axios.js

```diff
@@ -0,0 +1,5 @@
+var axios = require("axios");
+var axiosInstance = axios.create({
+  baseURL: "https://delphi.cs.uni-paderborn.de/api-legacy/features",
```

Comment:
Hard-coded URL again...

Reply:
It seems that you query the API directly from the frontend... Please consider changing that as we have discussed this several times already.

Reply:
It has been taken care by calling backend API which then calls Delphi WEB-API url. Thanks!

Reply:
Much better now... Thank you!